spec/controllers/profile_controller_spec.rb

```diff
@@ -52,4 +52,51 @@
       end
     end
   end
+
+  context 'setup mfa' do
+    it 'returns to profile if mfa is already setup' do
+      usermgr_stub_auth
+      get :setup_mfa
+      expect(response).to have_http_status(:redirect)
+      expect(subject).to redirect_to(profile_path)
+    end
+
+    it 'redirects to change mfa' do
```

Comment:
not sure what scenario this is testing, should it say something like
```suggestion
    it 'redirects to change mfa when cognito returns an error' do
```
?
and we probably also should be testing/stubbing when the MFA is set up response